index.js

```diff
@@ -43,7 +43,7 @@ exports.handler = function(event, context, callback) {
         if (imageProcessor && params) {
             ProcessImage.processImage(key, params, processCallback);
         } else if (rawProcessor) {
-            ProcessRaw.processRaw(processorData.path.split('/')[1], key.replace('/',''), processCallback);
+            ProcessRaw.processRaw(processorData.path, key.replace('/',''), processCallback);
```

Comment:
only replace starting forward slash. Do not replace if forward slash is down the path: use `replace(/^\//, "")` to replace. Also handle if used anywhere else.